pkg/pillar/diskmetrics/usage.go

```diff
@@ -241,3 +259,48 @@ func PersistUsageStat(log *base.LogObject) (*types.UsageStat, error) {
 	}
 	return usageStat, nil
 }
+
+// FoundLargeFiles is returned by FindLargeFiles
+type FoundLargeFiles struct {
+	Path string
+	Size int64
+}
+
+// FindLargeFiles walks a directory and reports all large files unless
+// they are in an excluded directory
+func FindLargeFiles(root string, limit int64, excludePaths []string) ([]FoundLargeFiles, error) {
```

Comment:
Can we rename this `FindFilesBySize`? Since we pass it the max size as a param, it isn't about large, as much as it is about "larger than X". And perhaps rename `limit` to `minSize` or similar?

Reply:
BySize implies you can specify a range or sizes or something.
But renaming limit to minSize makes sense (and updating the comment to say "larger than minSize".)
Let me know what you think.

Reply:
Looks good.